pkg/controller/expcertificates/issuing/issuing_controller_test.go

```diff
@@ -188,7 +188,17 @@ func TestIssuingController(t *testing.T) {
 							Message: "The certificate request failed because of reasons",
 						}),
 					)},
-				KubeObjects: []runtime.Object{},
+				KubeObjects: []runtime.Object{
+					&corev1.Secret{
```

Comment:
So if the currently managed certificate request is failed, and a next private key secret does not exist, what _do_ we do? Return nil and wait for the keymanager presumably? Perhaps keep this test case as-is and change the result, and then add an additional case that looks like this?